CHANGELOG-1.0.md

```diff
@@ -0,0 +1,11 @@
+# 1.0.x
+
+## Bug Fixes
+
+- API-599: fix const not supported by PHP 5.6
+
+# 1.0.1 (2018-04-05)
```

Comment:
In the 2.0 I did a single CHANGELOG. I think it's not necessary to do as the PIM and do a changelog for each version.